package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "6.12.0"
   },
   "dependencies": {
-    "@babel/core": "7.9.0",
```

Comment:
Please double check all changes to package.json and make sure you're only adding sentry and not changing anything else.

Reply:
Got it 👍🏾

Reply:
@znarf ,this has been resolved